### DISCUSSION TITLE
pycurl pausing download

### DISCUSSION THREAD

**Discussion by Agaulke2025** [curl/curl#11408] [Q&A]
Currently I'm trying to enable pausing downloads for functionality when the rate limit needs to change however I'm running into issues with setopt and the pause() method in pycurl. Initially just tried called c.pause(pycurl.PAUSE_RECV) that just ended with error code 43. I assumed this was because I never called setopt on PAUSE_RECV but can't find the documentation on the second argument that is required for this set opt: c.setopt(pycurl.PAUSE_RECV, ???). Not sure if this is the right approach for pause and start for file download. Any help would be appreciated, thanks!

**Comment by dfandrich** [comment]
This looks like a question better asked of the pycurl authors.
